metadatahandler.py

```diff
@@ -1,16 +1,80 @@
+import collections
+from enum import Enum
 import os
+import platform
+import util
 import shutil
-import collections
 import sys
-import platform
 import xml.etree.ElementTree as etree
 from xml.dom import minidom
-import util
 
 DosGame = collections.namedtuple('DosGame',
                                  'dosname metadataname name genres publisher developer year frontPic manual desc')
 
 
+class GameGenres(Enum):
+    # the target genres we want to map to
+    UNKNOWN             = "Unknown"
+    MISC                = "Misc"
+    ACTION_ADVENTURE    = "Action-Adventure"
+    ADVENTURE_VISUAL    = "Adventure-Visual"
+    ADVENTURE_TEXT      = "Adventure-Text"
+    BEATEMUP            = "BeatEmUp"
+    FPS                 = "Gun-FPS"
+    PINBALL             = "Pinball"
+    PLATFORM            = "Platform"
+    PUZZLE              = "Puzzle"
+    RACING              = "Racing"
+    RPG                 = "RPG"
+    SIMULATION          = "Simulation"
+    SHMUP               = "ShootEmUp"
+    SPORTS              = "Sports"
```

Comment:
While we're at it, can we change for singular 'Sport' ?

Reply:
sure!